gnmawsgr/tasks.py

```diff
@@ -259,7 +266,10 @@ def do_glacier_restore(request_id,itemid,path):
     while True:
         try:
             with open(filename,'wb') as fp:
-                key.get_file(fp, cb=partial(download_callback, rq), num_cb=40)
+                try:
+                    key.get_file(fp, cb=partial(download_callback, rq), num_cb=40)
+                except AttributeError:
+                    logger.error("No Key found in S3. Object values: {0}".format(rq.__dict__))
```

Comment:
I was not sure of the syntax for try so I looked it up. According to this (https://docs.python.org/2.7/tutorial/errors.html) after the code in except is run the programme contines by default.